File: tests/sample_files/test_pragma.py
def main(blacklist, white_list):  # blocklint: pragma
    # looking for master slave blocklint: pragma
    for item in blacklist:  # blocklint: some other tags pragma
        if item in white_list:  # blocklint: pragma
            slave = item  # blocklint: pragma
            assert slave == item


if __name__ == '__main__':
    main("", "")

File: tests/sample_files/test_pragma.cc
int test(std::vector<int> blacklist, int master){ // blocklint: pragma
    // looking for master/slave blocklint: pragma
    int slave = 0; // blocklint: some other tags pragma
    for(int i : blacklist) // blocklint: pragma
        if(i == master) // blocklint: pragma
            return slave;
}
